website/content/commands/server-install.mdx

```diff
@@ -52,6 +52,7 @@ you do not need to accept any terms.
 #### docker Options
 
 - `-docker-server-image=<string>` - Docker image for the Waypoint server.
+- `-docker-odr-image=<string>` - Docker image for the Waypoint Ondemand Runners
```

Comment:
Once we have more ODR types, it will probably make sense to make this a `type=image` map, but not a blocker now.

Reply:
Not sure what ya mean.

Reply:
I'm imagining a world where you have multiple kinds of on-demand runners, and wouldn't want to use the same docker image for each of them.

Reply:
I think for `server install` this is just setting up the initial config. Users can already setup multiple ODR pools (afaik) after the install.

Reply:
Yup, this is just for `server install` and the initial config.